app/components/Edit/EditSidebar.jsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withRouter } from 'react-router';
+
+class EditSidebar extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    const resource = this.props.resource;
+    let actionButtons = [
+      <button
+        className="edit--aside--content--button"
+        key="submit"
+        disabled={this.props.submitting}
+        onClick={this.props.handleSubmit}
+      >Save Changes</button>,
+      <button
```

Comment:
Trivial point, with our button styles more than anything, but on narrow screens this overflows messily: 

![screenshot 2018-03-25 00 46 49 2](https://user-images.githubusercontent.com/5233309/37872926-3306aaf0-2fc6-11e8-9467-ffe1ad264db9.png)

Creating #453 to track in general as a followup

Reply:
Nice. Thanks!